### PR TITLE
Submit project_one

### PR DESCRIPTION
**Pull request by CodingWithCDJE**
christopher.estevez@lc.cuny.edu

### CODE REVIEW

File: project_one/index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	<img src="Kappa_img.jpeg" id = "Main_img"> 
+</head>
+<body>
+
+	<script type="text/javascript" src= "index.js"></script>
+
+	<h1 id = "title"> WELCOME TO MY FIRST PROJECT </h1>
+
+	<div id= "First_Part">
+		<p id = "paragraph_one"> This is section one <button id = "grading-element-one"  onclick = "elementOne()"> Add Something </button> </p>
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected

File: project_one/index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+	<h1 id = "title"> WELCOME TO MY FIRST PROJECT </h1>
+
+	<div id= "First_Part">
+		<p id = "paragraph_one"> This is section one <button id = "grading-element-one"  onclick = "elementOne()"> Add Something </button> </p> 
+
+	</div>
+
+	<div id= "Second_Part">
+		<a href= "about.html" id= "grading-element-two" target = "_blank" > About Page </a>
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected

File: project_one/index.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+};
+
+
+const elementThree = () => {
+
+	let targ = document.getElementById("Third_Part");
+
+	let element_three_id = document.getElementById("grading-element-three");
+
+	let changes = document.createElement("P");
```
Comment (lehmanclass) [inline project_one/index.js]: not quite what i wanted, though this is done well.

Can you have the `p` tag with `id="grading-element-three"` _start_ in the DOM? Just use the click event to change `visibility: hidden` to `visibility: visible`

File: project_one/index.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+		<br></br>
+	<p> Once YOU CLICK YOU'RE DONE ... <button id = "grading-element-three" onclick = "elementThree()" > Click Here FOR MAGIC </button>
+	</p>
+
+
+	</div>
+
+	<div id = "Fourth_Part">
+
+	 <form name = "grading-element-four" method = "POST" action ="http://localhost:3000/form">
```
Comment (lehmanclass) [inline project_one/index.html]: `document.querySelector('form[name="grading-element-four"]).click()' does not submit the form...

Don't get me wrong, validation is great! But in this case i need the values to be pre-populated by your javascript...

File: project_one/index.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    </form>
+
+
+	</div>
+
+	<div id = "Fifth_Part">
+
+		<br> </br>
+
+		<p id= "grading-element-five"> LOOK AT ME I CHANGE COLOR ! </p>
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected

**Comment by lehmanclass** [comment]
just a few tweaks and you're there...can you add any extra credit in a `README.md` file?

**Comment by CodingWithCDJE** [comment]
Final commit, pushed changes.

**Pull request closed without merging**